lib/tasks/dfid.rake

```diff
@@ -0,0 +1,24 @@
+namespace :dfid do
+  desc "Migrate research outputs"
+  task migrate_research_outputs: :environment do
+    content_ids = Republisher.content_ids_for_document_type("dfid_research_output")
+    content_ids.each do |content_id|
```

Comment:
We should find a way of ordering this by the `updated_at` so the ordering doesn't get messed up... `.reverse` could be used if we can't find a better way

Reply:
I think this is resolved in d1dcef0.